### PR TITLE
Fixing heading issues with Openstack provider documentaiton.

### PR DESCRIPTION
**Pull request by elsonrodriguez**

### CODE REVIEW

File: docs/getting-started-guides/openstack-heat.md [hunk 1]
```diff
@@ -28,6 +28,7 @@ If you already have the required versions of the OpenStack CLI tools installed a
 
 ```sh
 sudo pip install -U --force 'python-openstackclient==2.4.0'
+sudo pip install -U --force 'python-heatclient==1.1.0'
```
Comment (kelseyhightower) [inline docs/getting-started-guides/openstack-heat.md]: I understand the fixing of the heading issues by adding whitespace, but did you really mean to add this line here?

Comment (elsonrodriguez) [inline docs/getting-started-guides/openstack-heat.md]: Yes, the openstackclient won't do heat things without the heatclient. I only caught it after doing a cleanroom test.

**Comment by elsonrodriguez** [comment]
@kelseyhightower Looking good?

**Comment by elsonrodriguez** [comment]
@johndmulhausen Looking good?

**Comment by pwittrock** [comment]
@elsonrodriguez Are you the maintainer of this doc, or is this a one-off fix?  I am looking for the proper reviewer.

According to the table in here:
http://kubernetes.io/docs/getting-started-guides/

This is maintained by @FujitsuEnablingSoftwareTechnologyGmbH, but I can't assign it to that user

**Comment by elsonrodriguez** [comment]
@pwittrock I re-wrote this documentation based on work from @zreigz, who is a member of  @FujitsuEnablingSoftwareTechnologyGmbH

**Comment by pwittrock** [comment]
@zreigz Can you comment on if this PR looks good?

**Comment by zreigz** [comment]
LGTM

**Comment by kelseyhightower** [comment]
LGTM

**Pull request merged**